Mastermind/Domain/BusinessRules/GamePlay.cs

```diff
@@ -0,0 +1,72 @@
+using Mastermind.Domain.Models;
+using Mastermind.Randomizer;
+
+namespace Mastermind.Domain.BusinessRules
+{
+    public class GamePlay
+    {
+        private readonly IRandomizer _randomizer;
+        
+        public GamePlay(IRandomizer randomizer)
+        {
+            _randomizer = randomizer;
+        }
+
+        public Game SetupGame()
+        {
+            return new Game
+            {
+                SelectedColours =_randomizer.GetRandomColours(GameConstants.SelectedNumberOfColours),
+                GuessingCount = 0,
+                GameState = GameStatus.Playing
+            };
+        }
+        
+        public void EvaluatePredictedAnswer(Game game)
+        {
+            UpdateGameLostStatus(game);
+            
+            game.GuessingCount++;
```

Comment:
Suggestion: One other approach here (and my personal preference) would be to have the entities mutate their own data where possible.

For example, `game` could have an `UpdateGuessingCount()` method which is called here. That way if the count is updated in other places, it makes it a bit easier to track what is calling it and update the implementation if necessary.